config.js

```diff
@@ -272,144 +265,75 @@ exports.updateFromArgv = function() {
-            exports.banned_users[user] = true;
-        }
-    }
-    if (argv.bansranked) {
-        for (const user of argv.bansranked.split(',')) {
-            exports.banned_users_ranked[user] = true;
-        }
-    }
-    if (argv.bansunranked) {
-        for (const user of argv.bansunranked.split(',')) {
-            exports.banned_users_unranked[user] = true;
-        }
-    }
-
-    if (argv.boardsizes) {
-        for (const boardsize of argv.boardsizes.split(',')) {
-            if (boardsize === "all") {
-                exports.allow_all_boardsizes = true;
-            } else {
-                exports.allowed_boardsizes[boardsize] = true;
-            }
-        }
-    }
-    if (argv.boardsizesranked) {
-        for (const boardsizeranked of argv.boardsizesranked.split(',')) {
-            if (boardsizeranked === "all") {
-                exports.allow_all_boardsizes_ranked = true;
-            } else {
-                exports.allowed_boardsizes_ranked[boardsizeranked] = true;
-            }
-        }
-    }
-    if (argv.boardsizesunranked) {
-        for (const boardsizeunranked of argv.boardsizesunranked.split(',')) {
-            if (boardsizeunranked === "all") {
-                exports.allow_all_boardsizes_unranked = true;
-            } else {
-                exports.allowed_boardsizes_unranked[boardsizeunranked] = true;
+    // r_u exports
+    /* 2) specific r_u cases:*/
+    for (const familyNameString of ["minrank", "maxrank"]) {
+        for (const argNameString of getArgNameStringsGRU(familyNameString)) {
+            if (argv[argNameString]) {
+                exports[argNameString] = parseRank(argv[argNameString]);
             }
         }
     }
 
-    if (argv.komis) {
-        for (const komi of argv.komis.split(',')) {
-            if (komi === "all") {
-                exports.allow_all_komis = true;
-            } else if (komi === "automatic") {
-                exports.allowed_komis[null] = true;
-            } else {
-                exports.allowed_komis[komi] = true;
-            }
-        }
-    }
-    if (argv.komisranked) {
-        for (const komiranked of argv.komisranked.split(',')) {
-            if (komiranked === "all") {
-                exports.allow_all_komis_ranked = true;
-            } else if (komiranked === "automatic") {
-                exports.allowed_komis_ranked[null] = true;
-            } else {
-                exports.allowed_komis_ranked[komiranked] = true;
-            }
-        }
-    }
-    if (argv.komisunranked) {
-        for (const komiunranked of argv.komisunranked.split(',')) {
-            if (komiunranked === "all") {
-                exports.allow_all_komis_unranked = true;
-            } else if (komiunranked === "automatic") {
-                exports.allowed_komis_unranked[null] = true;
-            } else {
-                exports.allowed_komis_unranked[komiunranked] = true;
+    for (const familyNameString of allowed_r_u_Families_numbers) {
```

Comment:
in #221 , these `allowed_r_u_Families_numbers` will be at the same position because we use them to filter the args we export (in summary in #221 we dont export any r_u arg (those which have a general arg, a ranked arg, and an unranked arg), the r_u args are exported in a different way after the general arg and ranked/unranked args are filter so that only `config.ranked[familyNameString]` and `config.unranked[familyNameString]` are exported (which are in all cases `config_r_u`))

Reply:
same as https://github.com/online-go/gtp2ogs/pull/230#discussion_r415144444